zebROS_ws/src/ros_control_boilerplate/config/2023_compbot_base_jetson.yaml

```diff
@@ -43,6 +43,7 @@ hardware_interface:
 
        - {name: four_bar_leader, type: can_talon_fx, can_bus: CANivore, can_id: 31, local: true}
        - {name: intake_leader, type: can_talon_fx, can_bus: CANivore, can_id: 41, local: true}
+       - {name: elevator_master, type: can_talon_fx, can_bus: CANivore, can_id: 51, local: true}
```

Comment:
Make this elevator_leader to match the other joints
Might as well add the elevator_follower as well
And then update can ids to match the robot
elevator is 31, 32
four bar is 41
intake is 51